torchvision/transforms/functional.py

```diff
@@ -35,6 +35,14 @@ def _is_numpy_image(img):
     return isinstance(img, np.ndarray) and (img.ndim in {2, 3})
 
 
+def _is_numpy(img):
+    return isinstance(img, np.ndarray)
+
+
+def _is_numpy_image_dim(img):
```

Comment:
Why did you name it `_is_numpy_image_dim`? If the goal of clarity extends to the code itself, wouldn't `_is_numpy_image` be clearer?

Reply:
I'm sorry I didn't notice your reply these days and thanks for your advice. 
Because the old code contains `_is_numpy_image `, I use a new name `_is_numpy_image_dim` for forward compatibility. Do you think this is unnecessary?

Reply:
I don't think its necessary to keep BC since it is a protected function.